### PR TITLE
MOD 9308- add the hero section to training video

### PR DESCRIPTION
**Pull request by chasls2**
**High level description:**

Add img and wording to the hero section of the training video page

**Technical details:**
Built out the component for the hero section
-added img
-add wording
-add css

**JIRA Ticket:**
[DEV-9308](https://federal-spending-transparency.atlassian.net/browse/DEV-9308)

**Mockup:**
https://app.zeplin.io/project/6307b25775991866c016769c/screen/63472dc7ee19fb538b35e6b8

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [x] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/_scss/core/_variables.scss [hunk 1]
```diff
@@ -92,6 +92,7 @@ $theme-color-3: $blue-warm-70v;
 $theme-color-4: $indigo-cool-80v;
 $theme-color-5: $blue-warm-5v;
 $theme-color-6: $blue-vivid-60v;
+$theme-color-7: #ccecf2;
```
Comment (nick-torres) [inline src/_scss/core/_variables.scss:95]: can you follow the convention of creating a variable for cyan-10: #ccecf2
then assigning theme-color-7: $cyan-10

File: src/_scss/pages/trainingVideos/_trainingVideosHeading.scss [hunk 2]
```diff
@@ -1,10 +1,98 @@
...
+    }
+
+    .training-videos__column-two {
+      display: flex;
+      align-items: center;
+      flex-direction: column;
+      justify-content: center;
+
+      @media (min-width: $tablet-screen) {
+        margin: 0 $global-margin * 15.5
```
Comment (nick-torres) [inline src/_scss/pages/trainingVideos/_trainingVideosHeading.scss:51]: missed semicolon

File: src/js/components/trainingVideos/trainingVideosHeading/TrainingVideosHeading.jsx [hunk 3]
```diff
@@ -4,12 +4,38 @@
...
-        </div>
+    <section className="training-videos__hero">
+        <FlexGridRow className="grid-content">
+            <FlexGridCol
+                desktop={6}
+                tablet={0}
+                mobile={0}>
+                <div className="training-videos__column-one">
+                    <div className="trianing-img-container">
+                        <img className="training-img" src="img/youtube-page/youtube-landing-page-hero-image@2x.webp" alt="placeholder" />
```
Comment (nick-torres) [inline src/js/components/trainingVideos/trainingVideosHeading/TrainingVideosHeading.jsx:20]: is alt=placeholder on purpose or do we have something to put there now

File: src/_scss/core/_variables.scss (hunk 1 above)
Comment (chasls2) [reply src/_scss/core/_variables.scss:95]: Good to know. Added

File: src/_scss/pages/trainingVideos/_trainingVideosHeading.scss (hunk 2 above)
Comment (chasls2) [reply src/_scss/pages/trainingVideos/_trainingVideosHeading.scss:51]: good catch! added

**Review by nick-torres — APPROVED** [review]

**Review by nick-torres — APPROVED** [review]

**Pull request merged**